.github/stale.yml

```diff
@@ -0,0 +1,65 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - 🚀Preview Release
+  - 🚀Production Release
+  - 🚀QA Preview Release
+  - 🚀Hot Fix Release
+  - 🚀QA ProductionRelease
+  - ⚕️NEEDS TRIAGE
+  - 🔬research  
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
Refactor to below:

```suggestion
  This issue has been automatically marked as stale because it has not had
  recent activity for 60 days. It will be closed after 7 days if no further activity occurs. Thank you
  for your contributions.
```